.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```

Comment:
Can we also add grouping of PR's based ? more details here - https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups. I can do a PR quickly to update, if you are okay with it.

Reply:
Hiya, thank you so much for this. Absolutely, will be great to see how would you group it, usually default is single PR which kind of makes it very self contained. Because sometimes API changes are breaking and they are hard to maintain and isolate when its a 1 PR grouped for many version update. ❤️☕️

What I meant by single PR and maintenance is kind of documented in the link shared above. But yes, please add the PR and will be good to see how that will group will benefit and the advantages. 

> By default, Dependabot raises a single pull request for each dependency that needs to be updated to a newer version. You can use groups to create sets of dependencies (per package manager), so that Dependabot opens a single pull request to update multiple dependencies at the same time.

Reply:
sounds good to me, let's keep it simple without grouping the PR for now and can tackle it in future if needed. Thank you.